packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Auth JS fundamentals
+---
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+This guide explains authentication fundamentals using Auth JS, known more formally as the Okta Auth JavaScript SDK, and provides a simple SPA application to demonstrate a sign-in use case.
+
+---
+
+**Learning outcomes**
+
+* Understand how to implement basic sign-in using the Okta Auth JavaScript SDK.
+* Understand basic installation and code configurations using the Okta Auth JavaScript SDK.
+* Implement the sample SPA use case and sign a user in to the application.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup/)
+* Basic knowledge of building front-end JavaScript applications
+
+**Sample code**
+
+* [static-spa sample](https://github.com/okta/okta-auth-js/tree/master/samples/generated/static-spa)
+
+---
+
+## About the Okta Auth JavaScript SDK
+
+The Okta Auth JS SDK builds on top of the [Authentication API](/docs/reference/api/authn/) and [OpenID Connect API](/docs/reference/api/oidc/) to enable you to create a fully branded sign-in experience using JavaScript.
```

Comment:
Technically, with the introduction of OIE, Auth JS SDK builds on top of IDX APIs as well, specifically the interaction code flow. I think we should mention that, since Authentication API is classic/old flow and the sample described here doesn't use it.

Reply:
Revised that sentence as: 

The Okta Auth JS SDK builds on top of the [Authentication API](/docs/reference/api/authn/) and [OpenID Connect API](/docs/reference/api/oidc/), as well as the Identity Engine [interaction code](/docs/concepts/interaction-code/) flow, to enable you to create a fully branded sign-in experience using JavaScript.